src/view/global.styl

```diff
@@ -1,6 +1,12 @@
+html, body {
```

Comment:
The extensions views were not filling the whole iframe. Because of that, tooltips that were placed above or below the trigger were getting cropped.